### PR TITLE
Update PandasTools.py to let LoadSDF be able to partial satinize molecules

### PR DESCRIPTION
**Pull request by e-mayo**
Add sanitize flags to LoadSDF function

### CODE REVIEW

File: rdkit/Chem/PandasTools.py [hunk 1]
```diff
@@ -440,54 +440,70 @@ def ChangeMoleculeRendering(frame=None, renderer='PNG'):
...
-  '''Read file in SDF format and return as Pandas data frame.
-    If embedProps=True all properties also get embedded in Mol objects in the molecule column.
-    If molColName=None molecules would not be present in resulting DataFrame (only properties
-    would be read).
-    '''
-  if isinstance(filename, str):
-    if filename.lower()[-3:] == ".gz":
-      import gzip
-      f = gzip.open(filename, "rb")
+            strictParsing=True, sanitize=False):
```
Comment (bp-kelley) [inline rdkit/Chem/PandasTools.py:449]: This needs to default to sanitize=True

File: rdkit/Chem/PandasTools.py [hunk 2]
```diff
@@ -440,54 +440,70 @@ def ChangeMoleculeRendering(frame=None, renderer='PNG'):
...
-    close()
-  RenderImagesInAllDataFrames(images=True)
-  return pd.DataFrame(records, index=indices)
+        f = filename
+        # don't close an open file that was passed in(molColName is not None)
+        close = None
+    records = []
+    indices = []
+    for i, mol in enumerate(
+        Chem.ForwardSDMolSupplier(f, sanitize=False, removeHs=removeHs,
```
Comment (bp-kelley) [inline rdkit/Chem/PandasTools.py:469]: You are hardcoding sanitize to False, here, I think you mean to use "sanitize"

File: rdkit/Chem/PandasTools.py [hunk 3]
```diff
@@ -440,54 +440,70 @@ def ChangeMoleculeRendering(frame=None, renderer='PNG'):
...
+                                  strictParsing=strictParsing)):
+        if not sanitize:
```
Comment (bp-kelley) [inline rdkit/Chem/PandasTools.py:471]: I'm confused by this logic, this calls SanitizeMol if sanitize==True, what is the intent?

**Comment by bp-kelley** [comment]
I'm a bit confuse what you mean by partial sanitization, but in any case, if sanitize=True you don't actually sanitize in this current implementation.

File: rdkit/Chem/PandasTools.py (hunk 3 above)
Comment (guiedd) [reply rdkit/Chem/PandasTools.py:471]: Yes it should say ```if sanitize:``` instead ```if not sanitize```
I think it's better let the user define whether sanitize or not, when loading an SDF file.

**Comment by greglandrum** [comment]
Closing due to lack of activity. Feel free to reopen if anyone has the time and inclination to revive this one.

**Pull request closed without merging**